columnflow/selection/cms/json_filter.py

```diff
@@ -43,9 +43,9 @@ def get_lumi_file_default(self, external_files: DotDict) -> str:
 def json_filter(
     self: Selector,
     events: ak.Array,
-    data_only=True,
+    data_only: bool = True,
     **kwargs,
-) -> ak.Array:
+) -> tuple[ak.Array, SelectionResult]:
```

Comment:
Can you also add the explanation in the `:return:` command in the doc string?

Reply:
Done 👍